wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_messages.scss

```diff
@@ -100,3 +100,24 @@
         padding-right: 3em;
     }
 }
+
+// make way for the navbar toggle icon when in mobile state
+@media screen and (max-width: $breakpoint-mobile) {
+    .success {
```

Comment:
Can we group these 3 together? Like ``
.success:before,
.error:before,
.warning:before
``